string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,15 @@
+/* exported isVowel */
+function isVowel(char) {
+  const upperCaseChar = char.toUpperCase();
+  if (
+    upperCaseChar === 'A' ||
+    upperCaseChar === 'E' ||
+    upperCaseChar === 'I' ||
+    upperCaseChar === 'O' ||
+    upperCaseChar === 'U'
+  ) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
This is totally fine! No need to change it, just want to show you another way of accomplishing this. This would be a perfect scenario where a `switch` statement would be useful.
```
function isVowel(character) {
  switch (character.toLowerCase()) {
    case 'a':
    case 'e':
    case 'i':
    case 'o':
    case 'u':
      return true;
    default:
      return false;
  }
}
```

Reply:
That makes sense! It's a lot more readable for sure. `switch` is still really new to me, so my brain doesn't go there first, but I'll look for opportunities to use it!